src/Package/Composer/ComposerDependencyChecker.php

```diff
@@ -30,8 +30,9 @@ public function getRequirements($workingDir)
         $requirements = array();
         if (isset($json['require'])) {
             foreach ($json['require'] as $packageName => $versionConstraint) {
-                if (preg_match('/.*\/.*/', $packageName)) {
-                    // NB: We're only concerned with package requirements not PHP/extension requirements
+                if ($packageName === 'php') {
```

Comment:
It might still be worth having the regex there so we don't add PHP extension requirements (e.g. `ext-gd`). They can be ignored for now, although perhaps in the future we could have checks for them too.